test/src/test/java/org/apache/accumulo/test/util/CertUtilsTest.java

```diff
@@ -126,26 +132,28 @@ public void publicOnlyVerfication() throws Exception {
     // this approximates the real life scenario. the client will only have the public key of each
     // cert (the root made by us as below, but the signed cert extracted by the SSL transport)
     CertUtils certUtils = getUtils();
-    File tempSubDir = tempDir.toPath().resolve(testName()).toFile();
-    assertTrue(tempSubDir.isDirectory() || tempSubDir.mkdir());
-    File rootKeyStoreFile = tempSubDir.toPath().resolve("root.jks").toFile();
-    certUtils.createSelfSignedCert(rootKeyStoreFile, "test", PASSWORD);
-    File publicRootKeyStoreFile = tempSubDir.toPath().resolve("publicroot.jks").toFile();
-    certUtils.createPublicCert(publicRootKeyStoreFile, "test", rootKeyStoreFile.getAbsolutePath(),
+    Path tempSubDir = tempDir.resolve(testName());
+    if (!Files.isDirectory(tempSubDir)) {
+      Files.createDirectories(tempSubDir);
+    }
```

Comment:
could add similar assertion: `assertTrue(Files.isDirectory(dir))` or move to util method I mentioned earlier